From 153a1fec386062ddd06e0c78938d1a64e0b70f20 Mon Sep 17 00:00:00 2001
From: DarthChungo <antonio@adaro.eu>
Date: Sat, 30 Dec 2023 16:03:10 +0100
Subject: [PATCH] st-patched: config

---
 config.def.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/config.def.h b/config.def.h
index fec62d7..0e485c9 100644
--- a/config.def.h
+++ b/config.def.h
@@ -73,7 +73,7 @@ static unsigned int cursorthickness = 2;
  *    Bold affects lines thickness if boxdraw_bold is not 0. Italic is ignored.
  * 0: disable (render all U25XX glyphs normally from the font).
  */
-const int boxdraw = 0;
-const int boxdraw_bold = 0;
+const int boxdraw = 1;
+const int boxdraw_bold = 1;
 
 /*
  * bell volume. It must be a value between -100 and 100. Use 0 for disabling
-- 
2.43.0

